extensions/amp-analytics/0.1/activity-impl.js

```diff
@@ -318,18 +318,22 @@ export class Activity {
     return this.activityHistory_.getTotalEngagedTime(secondsSinceStart);
   }
   /**
-   * Get the incremental engaged time since the last push and reset it.
+   * Get the incremental engaged time since the last push and reset it if asked.
    * @param {string} name
+   * @param {boolean} reset
```

Comment:
Should this be a `{string}` since is passed in as 'true'? Alternatively pass in as `true`?

Reply:
I changed it to string. IIRC the url replacement code doesn't like types that aren't strings.

Reply:
Thanks for catching that @elgreg. Hopefully we can get this reviewed and merged soon.

Reply:
My pleasure! We're really looking forward to using this in our implementation at Slate!